### PR TITLE
Added configuration for save db in cosv module

### PR DESCRIPTION
**Pull request by Cheshiriks**

### CODE REVIEW

File: save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entitiescosv/CosvFile.kt [hunk 1]
```diff
@@ -1,11 +1,13 @@
-package com.saveourtool.save.entities.cosv
+package com.saveourtool.save.entitiescosv
```
[the remark this answers is not in the document]
Comment (Cheshiriks) [reply save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entitiescosv/CosvFile.kt:1]: Entities that belong to different databases should be located in different directories for configuration

[the remark this answers is not in the document]
Comment (kgevorkyan) [reply save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entitiescosv/CosvFile.kt:1]: I think we already have different names, `entities.cosv` and `entities`, isnt it?

[the remark this answers is not in the document]
Comment (Cheshiriks) [reply save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entitiescosv/CosvFile.kt:1]: Then `entities.cosv` will be an internal directory for `entities`

[the remark this answers is not in the document]
Comment (kgevorkyan) [reply save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entitiescosv/CosvFile.kt:1]: Does this mean, that we will have 3 packages? `entities.cosv` `entitiescosv` and `entities`?

[the remark this answers is not in the document]
Comment (Cheshiriks) [reply save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entitiescosv/CosvFile.kt:1]: Now we will have two packages: `entitiescosv` and `entities`
But then we'll do three: `cosv.entities, `save.entities` and `common.entities`

[the remark this answers is not in the document]
Comment (akuleshov7) [reply save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entitiescosv/CosvFile.kt:1]: please add it to issue about renaming

**Review by akuleshov7 — APPROVED** [review]

File: save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entitiescosv/CosvFile.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (Cheshiriks) [reply save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entitiescosv/CosvFile.kt:1]: Yes. Added

**Pull request merged**